### ISSUE TITLE
Powerline/Nerd font glyphs not rendering properly

### ISSUE THREAD

**Issue by welnanick** [zeit/hyper#2201]
- [x] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: macOS Sierra 10.12.6
- **Hyper.app version**: 1.4.4 (Stable)
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: [hyper.js](https://gist.github.com/welnanick/9d3f1cc0ce8bf3ddc7cd89de4b48fef1) 
- **Relevant information from devtools** _(CMD+ALT+I on Mac OS, CTRL+SHIFT+I elsewhere)_: N/A 
- **The issue is reproducible in vanilla Hyper.app**: Yes

## Issue

Glyphs from a Powerline/Nerd fonts do not render with the correct line height in hyper. They render with some space at the top and bottom:
With Plugins:
<img width="406" alt="screen shot 2017-09-13 at 2 05 11 pm" src="https://user-images.githubusercontent.com/10217506/30400914-41f3d298-988d-11e7-9d9d-b3f5a9f0561f.png">
Vanilla:
<img width="401" alt="screen shot 2017-09-13 at 2 07 08 pm" src="https://user-images.githubusercontent.com/10217506/30400917-420acb42-988d-11e7-9b7a-2cd4cc6fac5e.png">

This is compared to the rendering from terminal.app with the same font settings:
<img width="391" alt="screen shot 2017-09-13 at 2 09 56 pm" src="https://user-images.githubusercontent.com/10217506/30400916-4206d4a6-988d-11e7-82b1-eac33beb0238.png">

This can be temporarily fixed using the developers tools if you remove "line-height" from the x-row css, but reverts when you restart hyper:
<img width="1373" alt="screen shot 2017-09-13 at 2 13 51 pm" src="https://user-images.githubusercontent.com/10217506/30401178-2266b552-988e-11e7-9ed7-b278a895fadc.png">

In addition some wider glyphs are cut off, you can see this in the circle from the previous images. This can only be fixed if you manually increase the width in the unicode-node css. I don't know if they are related issues or not.

**Comment by h93xV2** [comment]
This is really similar to the issue that I brought up in #2182. However, the way the error affects appearance seems to be different for our operating systems.

**Comment by haughters** [comment]
@welnanick Same thing here on Sierra, Hyper v1.4.6 with Menlo for Powerline. Your "line-height" temp solution gets it back to normal for me too.

**Comment by welnanick** [comment]
Issue is fixed in version 2.0.0 (Canary). I'm thinking the switch to xterm.js fixed this. I will keep this issue open until those changes are merged into stable.

**Comment by saschabratton** [comment]
If you are using stable, you can solve this, even after restart, by adding the following to your config:

```js
termCSS: `
  x-screen x-row {
    line-height: initial;
  }
`,
```
